projects/vdk-plugins/vdk-impala/tests/impala_error_handler_test.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import unittest
+from unittest.mock import call
+from unittest.mock import patch
+
+import impala
+from impala.error import HiveServer2Error
+from impala.error import OperationalError
+from vdk.plugin.impala.impala_error_handler import ImpalaErrorHandler
+from vdk.plugin.test_utils.util_funcs import populate_mock_managed_cursor
+
+
+@patch("time.sleep", return_value=None)
+class ImpalaErrorHandlerTest(unittest.TestCase):
+    def setUp(self):
+        self._query = "select 1"
+        self.error_handler = ImpalaErrorHandler(log=logging.getLogger())
+
+    # do not remove patched_time_sleep, it's used by @patch('time.sleep', return_value=None)
+    def test_default_retry(self, patched_time_sleep):
+        test_exception = OperationalError(
+            "Network error"
+        )  # we know for sure this is handled
+        (
+            mock_native_cursor,
```

Comment:
A formatting suggestion, would consider for readability purposes
```suggestion
        mock_native_cursor, _, _, \
        mock_recovery_cursor, _, _, _ = populate_mock_managed_cursor(
```
to focus left-to-right readers on interface surface utilised. 
I would rather simply
```suggestion
        mock_native_cursor, _, _, mock_recovery_cursor, _, _, _ = populate_mock_managed_cursor(
```